src/components/templates/Auditorium/components/Section/Section.scss

```diff
@@ -7,8 +7,6 @@ $section-seat-size: $avatarSize;
 $section-seat-size-min: $avatarSizeMin;
 $section-seat-spacing: 3px;
 
-$iframe-box-shadow: 2px 2px $spacing--lg $white;
-
```

Comment:
Hrmm.. still not sure this is right? The only file I expect to see any changes from on this PR should be in `src/scss/customOverrides.scss`?